### DISCUSSION TITLE
Tailwind vs vanilla CSS

### DISCUSSION THREAD

**Discussion by Sajithrajan03** [Abhinav-ark/timetable_csea#4] [Q&A] [answered]
Why do you prefer Vanilla CSS over Tailwind for HTML Projects?

**Comment by Abhinav-ark** [comment, marked as answer]
I use tailwind for react projects where i use components. In html since I repeat code, it's difficult to make a change if I use tailwind. I have to find and make changes in many places. If I use vanilla css, I can just modify the class which I've used in many places and the changes apply uniformly. However in react, since I don't repeat code, it's enough if I change tailwind classes in the component.
